tests/integration/multi_page/pages/.page1.py

```diff
@@ -0,0 +1,21 @@
+# This page should not be imported since it start with `.`
+
+import dash
+from dash import html, Input, Output, callback
+
+
+dash.register_page(__name__, id="page1")
```

Comment:
So the way we know this file isn't imported is that `page1` is already taken, and duplicating it would cause an error? I suppose that works, though it would be more obvious if this file directly threw an error, like:
```py
raise Exception("files starting with . should not be imported")
```